src/app/filesComponent/files-plugin.component.html

```diff
@@ -41,14 +41,14 @@ <h1><b>Please add data files to load...</b></h1>
                                             </select>
                                         </div>
                                     </div>
-                                    <div id="ambiguities-row" class="form-group row">
+                                    <div id="ambiguities-row" *ngIf="SelectedDefaultDistanceMetricVariable=='tn93'" class="form-group row">
```

Comment:
Was this an element that slide up/down to show/hide?  If it was we should stick to the slide up/down animation.  If not, then you can ignore this.

Reply:
The current deployed dev site slides up/down which does look better than mossy site. I think I was trying to get the file settings and global setting distance metrics to sync but forgot to revisit it and finish fixing the bug. Currently on both dev and mossy, updates of distance metric and link threshold variables on file setting updates global settings but updates of these variable on global settings doesn't update file setting. Please let me know how I should resolve this.

Reply:
Got it, yes updates should be 2-way.